pir/cpp/payload.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package pir;
+
+message Payload {
+  repeated bytes query = 1;
```

Comment:
Is this the response to a query or the query itself? If the former, then this should be something like result or reply. Please also add a description of what this field is.